docs/user-guide/creating-application/deployment-template.md

```diff
@@ -20,32 +32,201 @@ You can select a default deployment chart from the following options:
 3. [Job & CronJob](deployment-template/job-and-cronjob.md)
 4. [StatefulSet](deployment-template/statefulset.md)
 
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/deployment-template/select-devtron-chart.gif)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/deployment-template/deployment-chart.png)
+### From Custom Charts
```

Comment:
There are some limitations in the system if you use custom chart, not sure if we have mentioned you somewhere or not

Reply:
Will be pickup in the another PR dedicated to custom charts